code-of-conduct.md

```diff
@@ -72,16 +72,18 @@ Conduct may be permanently removed from the project team.
 
 For incidents occuring in the Kubernetes community, contact the [Kubernetes Code of Conduct Committee](https://git.k8s.io/community/committee-code-of-conduct) via <conduct@kubernetes.io>. You can expect a response within three business days.
 
-For other projects, please contact the CNCF staff via <conduct@cncf.io>. You can expect a response within three business days. 
+For other projects, or for incidents that are project-agnostic or impact multiple CNCF projects, please contact the [CNCF Code of Conduct Committee](https://www.cncf.io/conduct/committee/) via conduct@cncf.io.  Alternatively, you can contact any of the individual members of the [CNCF Code of Conduct Committee](https://www.cncf.io/conduct/committee/) to submit your report. For more detailed instructions on how to submit a report, including how to submit a report anonymously, please see our [Incident Resolution Procedures](https://www.cncf.io/conduct/procedures/). You can expect a response within three business days.
 
 In matters that require an outside mediator, CNCF has retained Mishi Choudhary (mishi@linux.com). Use of an outside mediator can be requested when reporting or used at CNCF staff's discretion. In general, contacting <conduct@cncf.io> directly is preferred.
 
 
 ## Enforcement 
 
-The Kubernetes project's [Code of Conduct Committee](https://github.com/kubernetes/community/tree/master/committee-code-of-conduct) enforces code of conduct issues. For all other projects, the CNCF enforces code of conduct issues.
+The Kubernetes project's [Code of Conduct Committee](https://github.com/kubernetes/community/tree/master/committee-code-of-conduct) enforces code of conduct issues.
 
-Both bodies try to resolve incidents without punishment, but may remove people from the project or CNCF communities at their discretion. 
+For all projects that do not have their own Code of Conduct Committee or other Code of Conduct incident responders, and for incidents that are project-agnostic or impact multiple projects, the CNCF Code of Conduct Committee enforces code of conduct issues. For more information, see our [Jurisdiction and Escalation Policy](https://www.cncf.io/conduct/jurisdiction/)
```

Comment:
Small nit (sorry for not catching sooner): ..."the CNCF Code of Conduct Committee **resolves** code of conduct **incidents**."  - i.e., replace "enforces" with "resolves" and replace "issues" with "incidents"

Reply:
@RichiH , good suggestion.  Let's revisit at the CoC Working Group phase (step 2 in the [overall process](https://www.cncf.io/blog/2022/06/06/upcoming-code-of-conduct-updates-at-cncf/)).  I am thinking we might want to link to a separate page that lists projects with their own CoC Committees, since projects that form their own CoC response teams may change over time, and if the number grows, we don't want to clutter the Code of Conduct itself with a long list.

Reply:
> Does it make sense to list the general one first and then say
> 
> "Projects may choose to have their own CoCCs. The list of CNCF projects which currently have their own CoCC is a follows:
> 
> * K8s
> 
> "?

@RichiH , good suggestion. Let's revisit at the CoC Working Group phase (step 2 in the [overall process](https://www.cncf.io/blog/2022/06/06/upcoming-code-of-conduct-updates-at-cncf/)). I am thinking we might want to link to a separate page that lists projects with their own CoC Committees, since projects that form their own CoC response teams may change over time, and if the number grows, we don't want to clutter the Code of Conduct itself with a long list.